### DISCUSSION TITLE
Problem in Chrome

### DISCUSSION THREAD

**Discussion by julian-studio-x** [google/model-viewer#3574] [Q&A] [answered]
Overall bug in chrome, dont see any viewer anymore.
My url: https://vr.studio-x.nl/AR/bouwschool-twente
Your url: https://modelviewer.dev/examples/loading/
![Schermafbeelding 2022-07-05 150058](https://user-images.githubusercontent.com/93528885/177333619-b4a25c93-d351-4bce-9a31-117ddb7ca8b4.png)

**Comment by elalish** [comment, marked as answer]
Your URL is working fine for me in Chrome. Can you give more details about how to reproduce this? What device? Does it work anywhere else? Are you saying modelviewer.dev was also not loading properly for you?

**Reply by julian-studio-x** [reply]
HI Elalish, thanks for replying! I've updated chrome and now it works fine again! Every website that used WebGL crashed for me. 

Have a great day!

^Julian
